plugins/woocommerce/tests/php/src/Caching/ObjectCacheTest.php

```diff
@@ -55,19 +55,6 @@ protected function get_from_datastore( $id ) {
 		$this->sut->flush();
 	}
 
-	/**
-	 * Runs after each test.
-	 */
-	public function tearDown(): void {
-		delete_option( 'wp_object_cache_key_prefix_the_type' );
-		remove_all_filters( 'wc_object_cache_get_engine' );
```

Comment:
This filter is still being added in`test_custom_cache_engine_via_hook` and therefore should be removed in `tearDown` here -- or am I missing something?